p2p/libp2p/discovery/optimizedKadDhtDiscoverer.go

```diff
@@ -231,7 +238,7 @@ func (okdd *optimizedKadDhtDiscoverer) findPeers(ctx context.Context) {
 
 	err := okdd.kadDHT.Bootstrap(ctx)
 	if err != nil {
-		okdd.log.Debug("kad dht bootstrap", "error", err)
+		okdd.log.Debug("kad dht bootstrap", "error", err, "network", okdd.networkType)
```

Comment:
valid for all prints: can we put the network as the first parameter or in the log message for easier filtering when searching through the logs?

Reply:
updated